src/index.css

```diff
@@ -2,3 +2,5 @@ body {
   margin: 0;
   padding: 0;
 }
+
+.MuiPaper-root-8 .MuiCardContent-root-67:last-child { padding-bottom: 16px; }
```

Comment:
Não é bom confiar nestes nomes de classe pois eles são dinâmicos, se o layout do componente for alterado, o CSS vai parar de funcionar. Tem um exemplo de CSS in JS no Material UI neste link: https://material-ui-1dab0.firebaseapp.com/layout/css-in-js/ .

Na documentação do `<Typography />` (aqui: https://material-ui-1dab0.firebaseapp.com/style/typography/) tem exemplo de algumas propriedades envolvendo margin ou padding, não lembro bem. Também estou me familiarizando com o framework. Talvez seja uma opção mais clean.